.pre-commit-config.yaml

```diff
@@ -0,0 +1,58 @@
+minimum_pre_commit_version: "2.9.2"
+repos:
+  - repo: meta
+    hooks:
+      - id: identity
+      - id: check-hooks-apply
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: "" # Can not be removed, so leave this empty.
```

Comment:
I guess we want to just use the latest version 🤔  Does `master` work as a revision?

Reply:
ok, fair enough!